isyntax2raw/cli/isyntax2raw.py

```diff
@@ -77,7 +77,19 @@ def write_tiles(
         wt.write_pyramid()
 
 
-cli.add_command(write_tiles, name='write_tiles')
+@cli.command(name='write_metadata')

```

Comment:
Did we need to do this vs. just using `@click.command()`?

Reply:
https://click.palletsprojects.com/en/8.0.x/commands/

https://github.com/pallets/click/issues/1123